README.md

```diff
@@ -146,7 +146,7 @@ playbook to install KubeVirt from given manifest.
 | ---------------- | ------------- | -------------------------------------- |
 | kubevirt\_mf     | string | Path to KubeVirt manifest, you can get one from [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources |
 | openshift\_ansible\_dir | string | Path to [OpenShift Ansible project][openshift-ansible-project] |
-| kubeconfig | string | Path to kubeconfig |
+| kubeconfig | string | Path to kubeconfig containing authentication information for desired cluster. |
```

Comment:
@lukas-bednar I still don't follow where kubeconfig comes from. Where does one can find that file? And I would replace `kubeconfig` in description (that's why it's called description) with `the KubeVirt|Kubernetes(?) configuration file`.

Reply:
@alexxa https://docs.openshift.com/container-platform/3.7/cli_reference/manage_cli_profiles.html

Reply:
@gbenhaim Good! So, I would put it as: 

Path to the [`~/.kube/config`](https://docs.openshift.com/container-platform/3.7/cli_reference/manage_cli_profiles.html#switching-between-cli-profiles) file containing authentication information for desired cluster

Reply:
@lukas-bednar ^^

Reply:
Addressed